src/components/AppComponents/AppHeader/AppHeader.tsx

```diff
@@ -92,6 +106,19 @@ const AppHeader: FC<HeaderProps> = () => {
                 }}
                 data-testid="clerk-user-button"
               />
+            ) : (
+              selectedArea == siteAreas.teachers && (
+                <SignUpButton forceRedirectUrl={onboardingRedirectUrl}>
+                  <OakSecondaryButton
+                    name="sign up"
```

Comment:
The name attribute here isn't necessary as there is no form element associated with this button and the span inside the button element has text which will be used for the screenreader.